.github/CONTRIBUTING.md

```diff
@@ -63,6 +63,8 @@ The  `stable-` prefixed branches are what’s actually deployed. When releasing
 
 For maintainers.
 
-* Make changes as necessary and merge into master. **The changes must include updated release notes with a new version number**, as this is automatically used as the Nuget version when deploying.
-* Create a PR from `master` to the latest `stable-` branch
-* Merge if the tests pass.
+* Make necessary changes to the code
+* Update the changelog
+* Update the version and release notes in the fsproj file
+* Commit and tag the commit (this is what triggers deployment from  AppVeyor). For consistency, the tag should be identical to the version  (e.g. `1.2.3`).
```

Comment:
I recommend having the tag be slightly different from the version.  If the version is is `1.2.3`, then the tag would be something like `v/1.2.3`.  That way, tags related to versioning can be grouped together and visually collapsed in some GUI.

As an example, here are some of the lexicographically-first tags in the repository for my project at work (as seen via [SourceTree](https://www.sourcetreeapp.com/)).
![Screenshot 2019-09-26 08 32 41](https://user-images.githubusercontent.com/34664007/65692607-74919880-e038-11e9-8991-095f7a07023b.png)

All commits into `develop` are merge commits, and every one of these commits is automatically built, tagged, and includes published artifacts.  Of course, we would use all of these tags to quickly go from running software back to the corresponding commit when something is found that we want to change (like a bug).  Since our project is still so new, we haven't done much of this yet.  Similarly, these are the only type of tags so far (i.e. no manually created tags and no other automatically created tags).

![Screenshot 2019-09-26 08 44 36](https://user-images.githubusercontent.com/34664007/65694250-25993280-e03b-11e9-8579-f9996e8b7ede.png)

In the past, my experience is that the product owner wants to tag commits corresponding to any version they presented to others.  It can be a demo to higher ups within their own company or people outside their company.  Those tags might look like `demo/2019-09-26_CEO` if the project owner demoed the application to the CEO of their company on 2019-09-26.  Since it starts with `demo/`, these too are grouped together and collapsable.

Of course these specific use cases probably don't apply to Elmish.WPF, but it is very difficult to rename tags later, so I wanted to share this idea now.

Reply:
Thanks! I have been very happy with just the version for my other projects, and it's also what's used by other large projects such as [FAKE](https://github.com/fsharp/FAKE/releases) and [Paket](https://github.com/fsprojects/Paket/releases).

That said, I don't have a problem with your suggestion. Consistency is the most important thing.

I've never seen the `v/` prefix though. I have seen `v`, but no slash. Seems tooling-specific whether that will help or not.

I'm slightly inclined to just to with `1.2.3`, but if you want to insist on `v1.2.3` or `v/1.2.3` or something else, we can decide on that instead.

Reply:
Not insisting.  Whatever you think is best is fine with me.  I just wanted to share my experience in case that would change your mind.

Reply:
Thanks for your input! Will leave this unresolved and think/research a bit more.